tests/share/normalize/test_links.py

```diff
@@ -1,8 +1,35 @@
 import pytest
 import rfc3987
 
-from share.normalize.links import DOILink
-from share.normalize.links import OrcidLink
+import calendar
+
+from share.normalize.links import DOILink, OrcidLink, DateParserLink
+
+
+@pytest.mark.parametrize('date, result', [
+    (None, None),
+    ('0059-11-01T00:00:00Z', ValueError('0059-11-01T00:00:00Z is before the lower bound 1200-01-01T00:00:00+00:00.')),
```

Comment:
Could you add a test case or two for coercing other time zones into UTC?
Maybe one for an invalid time zone as well?